### PR TITLE
Added a couple of minor fixes based on user feedback

### PR DESCRIPTION
**Pull request by Quexington**

### CODE REVIEW

File: docs/coins_spends_and_wallets.md [hunk 1]
```diff
@@ -239,6 +239,8 @@ Before we go there, let's explain signatures:
 
 ## BLS Aggregated Signatures
 
+If you don't have a fundamental understanding of cryptographic signatures, it will be good to [take a look at that](https://en.wikipedia.org/wiki/Digital_signature) before you continue.
```
Comment (hoffmang9) [inline docs/coins_spends_and_wallets.md:242]: how about "it will be good to familiarize yourself with [digital signatures] ... before you continue."

Comment (Quexington) [reply docs/coins_spends_and_wallets.md]: How does it look now?

Comment (hoffmang9) [reply docs/coins_spends_and_wallets.md]: "Take a look at that" is generally considered a bad link since you are linking to a text on "digital signatures"

**Review by hoffmang9 — APPROVED** [review]
ok

**Pull request merged**